commands/show-beacons.go

```diff
@@ -43,7 +43,10 @@ func showBeacons(c *cli.Context) error {
 	res.DB.SelectDB(c.String("database"))
 
 	var data []tbdData.TBDAnalysisView
-	TBD.GetTBDResultsView(res, cutoffScore).All(data)
+
+	ssn := res.DB.Session.Copy()
+	TBD.GetTBDResultsView(res, ssn, cutoffScore).All(&data)
```

Comment:
Oddly enough you need to pass arrays by reference to have them filled by the iterator.